sardana_limaccd/ctrl/LimaCCDCtrl.py

```diff
@@ -473,6 +473,16 @@ def GetCtrlPar(self, parameter):
 #                Axis Extra Attribute Methods
 ###############################################################################
 
+    def GetAxisPar(self, axis, parameter):
+        if parameter == "value_ref_pattern":
+            return self._value_ref_pattern
+        elif parameter == "value_ref_enabled":
+            return self._value_ref_enabled
+        elif parameter == "shape":
+            data = self._limaccd.read_attributes(("image_width", "image_height"))
+            return data[0].value, data[1].value
+        return super().GetAxisPar(axis, parameter)
```

Comment:
This is not python2 compatible. Could we use this instead?:
`return super(LimaCCDTwoDController, self).GetAxisPar(axis, parameter)`

Reply:
Yes, sure! Thanks for pointing it out.